espnet2/enh/separator/ineube_separator.py

```diff
@@ -0,0 +1,147 @@
+import torch
+import asteroid_filterbanks.transforms as af_transforms
+from collections import OrderedDict
+from typing import List
+from typing import Tuple
+from typing import Union
+from torch_complex.tensor import ComplexTensor
+from espnet2.enh.separator.abs_separator import AbsSeparator
+from asteroid.dsp.beamforming import condition_scm
+from asteroid_filterbanks import make_enc_dec
+from asteroid.utils.torch_utils import pad_x_to_y
+from espnet2.enh.layers.tcndenseunet import TCNDenseUNet
+
+
+class iNeuBe(AbsSeparator):
+    def __init__(
+        self,
+        n_spk=1,
+        n_fft=512,
+        stride=128,
+        window="hanning",
+        mic_channels=1,
+        hid_chans=32,
+        hid_chans_dense=32,
+        ksz_dense=(3, 3),
+        ksz_tcn=3,
+        tcn_repeats=4,
+        tcn_blocks=7,
+        tcn_channels=384,
+        activation=torch.nn.ELU,
```

Comment:
It is better to use a string for choosing different activation functions.
Because in ESPnet, arguments are parsed directly from the yaml file. Strings are better supported than language-specific object.

Reply:
I added an util for doing that https://github.com/popcornell/espnet/blob/177fc0d2eabbdbe531f5289437991603325a52fe/espnet2/torch_utils/get_layer_from_string.py#L5